crates/examples/combined/orchestrator.rs

```diff
@@ -24,7 +24,13 @@ pub mod infra;
 async fn main() {
     setup_logging();
     setup_backtrace();
-    let args = OrchestratorArgs::parse();
-
-    run_orchestrator::<TestTypes, DANetwork, QuorumNetwork, NodeImpl>(args).await;
+    let config = read_orchestrator_init_config::<TestTypes>();
+    let orchestrator_url = Url::parse("http://localhost:4444").unwrap();
```

Comment:
Can we make it so we set the orchestrator url to something we pass in as arguments?  We may need to change the port sometimes when we run on AWS.  We can have it default to localhost:4444 if no arguments are provided though, since having a default will make it more ergonomic.  This is the only change request I have for the PR.